### PR TITLE
Python black

### PR DESCRIPTION
**Pull request by uberspot**
I just ran `black -l 110 -t py37 --exclude reclass .` to enforce the same formatting on all kapitan source code for easier reviewing/coding in the future. 

https://pypi.org/project/black/

### CODE REVIEW

**Comment by ramaro** [comment]
We should make this part of the pipeline

**Comment by uberspot** [comment]
> We should make this part of the pipeline

Done :)

File: kapitan/inputs/helm/cffi_build.py [hunk 1]
```diff
@@ -5,18 +5,17 @@ def main():
...
         void free(void* ptr);
-    """)
-
-    ffi.set_source(
-        "helm_binding",  # specify name for importing this module
-        None
+    """
     )
 
+    ffi.set_source("helm_binding", None)  # specify name for importing this module
```
Comment (gburiola) [inline kapitan/inputs/helm/cffi_build.py]: interesting to see how black reformatted this block. before, the comment was in the middle of the parameters.

File: kapitan/lint.py [hunk 2]
```diff
@@ -56,7 +56,15 @@
 """
 
 
-def start_lint(fail_on_warning, skip_class_checks, skip_yamllint, inventory_path, search_secrets, secrets_path, compiled_path):
+def start_lint(
```
Comment (gburiola) [inline kapitan/lint.py]: sooo much easier to read

Comment (uberspot) [reply kapitan/lint.py]: I know right :D

**Pull request merged**